src/main/resources/templates/editProfile.html

```diff
@@ -12,7 +12,8 @@
     <title>Edit Profile</title>
 </head>
 <body>
-    <div id="nav-placeholder"></div><br><br>
+<div id="nav-placeholder"></div>
+<div id="footer-placeholder"></div>
```

Comment:
is the footer really all the way down? why is it right under the nav within the html if it should be the last thing on the page?

Reply:
sorry